✨ Feat : 스플래시, 메인 로그인, 404

hanhj0516 opened the pull request:
메인 로그인 마크업 완료

> ### 잠깐! PR하기 전에 확인해주세요!

✅ PR 제목의 형식을 잘 작성했나요?(커밋메시지 컨벤션과 동일)   
✅ 불필요한 코드를 제거했나요?   
✅ 작업 시작 전 후로 develop브랜치가 최신상태임을 확인했나요?   
✅ 라벨은 등록했나요?   
<br>

✨ PR 한 줄 요약
-------------------------------------------------
메인 로그인 화면 마크업 완료
스플래시, 404 진행 중

<br>
<br>

🛠 상세 작업 내용
-------------------------------------------------  
<!--여기에 작성해주세요-->

<br>
<br>

📸 참고 사항
-------------------------------------------------
<!--여기에 작성해주세요-->

<br>
<br>


💡 관련 이슈
-------------------------------------------------
<!--여기에 작성해주세요-->

<br>
<br>

hanhj0516 committed:
✨ Feat : 스플래시, 메인 로그인, 404
메인 로그인 마크업 완료

yujin1228 commented on src/pages/LoginPage/MainLoginPage.style.js:
```diff
@@ -0,0 +1,118 @@
+    width: 212px;
+  }
+`
+export const LoginContainer = styled.form`
```
특별히 form태그를 쓰신 이유가 있을까요?? 발생할 submit 이벤트가 없는 페이지라 그냥 div 또는 section 태그를 사용해도 무방할것같습니다!

hanhj0516 replied:
수정하겠습니당!

yujin1228 commented on src/pages/LoginPage/MainLoginPage.jsx:
<img width="507" alt="image" src="https://github.com/FRONTENDSCHOOL7/final-05-oguogu/assets/53458437/36540589-9a6d-45f7-a7d0-a4666ff4d71f">

이 페이지를 실행해보면 높이가 820 이상인 스크린에서 봤을때 밑부분이 남는 현상이 보입니다
이 부분 스타일 수정 부탁드려요!

hanhj0516 replied:
수정하겠습니당!

yujin1228 commented on src/pages/LoginPage/MainLoginPage.style.js:
```diff
@@ -0,0 +1,118 @@
+`
+
+export const Join = styled.button`
+  font-family: inherit;
```
글로벌스타일에서 버튼태그에 font-family: main-font 로 되어있기때문에 이부분은 없어도 무방합니다!
또 버튼은 폰트크기가 상속되지않아요! 비밀번호찾기 와 회원가입 폰트가 현재 다른것으로 보입니다. 폰트크기 설정이 필요합니다.

hanhj0516 replied:
수정하겠습니당!

yujin1228 commented on src/pages/LoginPage/MainLoginPage.style.js:
```diff
@@ -0,0 +1,118 @@
+  font-size: 12px;
+`
+
+export const FindPassword = styled.a`
```
비밀번호찾기는 a태그이고 회원가입은 버튼태그인 이유가 무엇인가요?

hanhj0516 replied:
원래는 비밀번호 찾기는 다른 페이지에서 할 거라고 생각해서 한거긴한데 수정하겠습니다!

hanhj0516 committed:
🎨 Style : 수정 요청 사항 수정 완료
2023.10.21 수정 요청 사항들 수정했습니다.

hanhj0516 committed:
Merge branch 'develop' into feature/splash-mainLogin-hyeji

hanhj0516 committed:
✨ Feat : 스플래시,메인 로그인 전환

- 스플래시에서 딜레이 후 메인 로그인으로 화면 전환
- 메인 로그인에서 슬로건, 로그인 컨테이너 애니메이션 적용

hanhj0516 committed:
✨ Feat : 404 페이지 구현 완료

- 각 페이지에서 새로고침하면 스플래시가 계속 뜨는 문제 해결
- 없는 주소면 404페이지 뜨기
- 이전 페이지 버튼 누르면 바로 이전에 있던 페이지로 돌아가기

yujin1228 reviewed (approved):
굿
